MergerLogic/ImageProcessing/TileMerger.cs

```diff
@@ -18,9 +18,21 @@ public TileMerger(ITileScaler tileScaler, ILogger<TileMerger> logger)
             this._tileScaler = tileScaler;
         }
 
-        public Tile? MergeTiles(List<CorrespondingTileBuilder> tiles, Coord targetCoords, TileFormatStrategy strategy)
+        public Tile? MergeTiles(List<CorrespondingTileBuilder> tiles, Coord targetCoords, TileFormatStrategy strategy, bool uploadOnly = false)
         {
-            var images = this.GetImageList(tiles, targetCoords);
+            if(uploadOnly) {
+                // Ignore target if in upload only mode
+                tiles = tiles.Skip(1).ToList();
+
+                // In case there is only one source then we can just return the data as is
+                if(tiles.Count == 1) {
+                    Tile? rawTile = tiles[0]();
+                    rawTile?.ConvertToFormat(strategy.ApplyStrategy(rawTile.Format));
+                    return rawTile;
```

Comment:
Add appropriate debug log same as below
"this._logger.LogDebug($"[{MethodBase.GetCurrentMethod().Name}] 1 images found");"
or "this._logger.LogDebug($"[{MethodBase.GetCurrentMethod().Name}] No images where found return null");"

Reply:
Added.